src/RealtimeEffectPanel.cpp

```diff
@@ -267,7 +267,7 @@ namespace
       RealtimeEffectState* mEffectState{nullptr};
 
       ThemedButtonWrapper<wxButton>* mChangeButton{nullptr};
-      wxToggleButton* mEnableButton{nullptr};
+      wxButton* mEnableButton{nullptr};
```

Comment:
wxBitmapToggleButton exists.  Is it the right thing for you?

Reply:
Using `wx*ToggleButton` will simplify logic, but it looks noticeably different compared to `wx*Button` in GTK. Few lines of code should be added later to change icon when effect changes it's power state.